README.md

```diff
@@ -1,5 +1,7 @@
 # µgo
 
+[![Build Status](https://travis-ci.org/googlesamples/firebase-arduino.svg?branch=master)](https://travis-ci.org/googlesamples/firebase-arduino)
```

Comment:
Looks like a copy paste error.

This repo is `googlesamples/mugo`, not `googlesamples/firebase-arduino`.